### DISCUSSION TITLE
Farm annual energy production

### DISCUSSION THREAD

**Discussion by MYMahfouz** [NatLabRockies/floris#550] [Q&A] [answered]
Hi ,

I have two separate questions about AEP calculation:

1. I recently switched from FLORIS v2.4, and I see that for the same farm the AEP calculated in the newer FLORIS version (v3) is smaller. Is this normal/expected? (I am using default gauss wake model inputs in both cases)

2. If i use the .get_farm_AEP() for one wind direction it takes the same time it takes if I am calculating all wind directions in v3. I believe for a single direction AEP calculation the older FLORIS version (v2.4) was faster. Is this true?

Thanks in advance.
Youssef

**Comment by MYMahfouz** [comment, marked as answer]
1. I found the reason for the AEP difference. I was using 5 grid points in v2.4 and only 3 grid points maximum are allowed for v3.

**Reply by paulf81** [reply]
Thank you for posting the solution!
